doc/source/deploy-guide/prepare-localhost.rst

```diff
@@ -182,11 +182,7 @@ floating network (you only have to do it once):
 
 .. code-block:: console
 
-   openstack network create ${PREFIX}-net
-   openstack subnet create --network ${PREFIX}-net --subnet-range 192.168.100.0/24 ${PREFIX}-subnet
-   openstack router create ${PREFIX}-router
-   openstack router set --external-gateway floating ${PREFIX}-router
-   openstack router add subnet ${PREFIX}-router ${PREFIX}-subnet
+   ./run.sh deploy_network
```

Comment:
This is fine, but it hides the necessary input for deploy_network (the need to pass prefix for example). How do you pass prefix btw?

Reply:
WIth the heat parameter called prefix and then I set this in playbooks/openstack-deploy_network.yml via parameters

Reply:
I totally missed that L21. Cool thanks.